### PR TITLE
Add math writing, fix codechat

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

File: source/custom/problemsetguide/pset-intro.ptx [hunk 1]
```diff
@@ -162,10 +162,11 @@
...
-        <image xml:id="writingguide" source="../../../linear-algebra/assets/custom/Su-MathWriting.png" width="100%">
+        <image xml:id="writingguide" source="custom/Su-MathWriting.png" width="100%">
             <description>
                 <p>
-                    This is a short guide to writing mathematics well by Francis Su (Harvey Mudd College). 
+                    This is a short guide to writing mathematics well by Francis Su (Harvey Mudd College). A
+                    <dataurl source="custom/Su-MathWriting.pdf">PDF</dataurl> is also avaialble. 
                     Use it a checklist when writing your solutions. <alert>Almost every deduction we give for communication is a result of work that violates at least one of these principles</alert>.
                 </p>
             </description>
```
Comment (StevenClontz) [inline source/custom/problemsetguide/pset-intro.ptx:172]: I've added a PDF of this document that students can access. Note that this description is usually hidden behind a little "info" icon, so students may not read it; maybe consider moving it into a separate paragraph?

File: codechat_config_cal_custom.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Replace the contents of `codechat_config.yaml` with this file to edit Calculus
+output_path: output/cal-web-custom
+args: "{sys_executable} -m pretext build cal-web-custom --xmlid {xml_id} --no-knowls"
+project_type: PreTeXt
```
Comment (StevenClontz) [inline codechat_config_cal_custom.yaml:4]: I think these custom codechat config files should get your codechat working - don't forget to paste them into the main codechat file when working on one book vs another...

File: source/custom/problemsetguide/pset-intro.ptx (hunk 1 above)
Comment (jkostiuk) [reply source/custom/problemsetguide/pset-intro.ptx:172]: thank you!

**Review by jkostiuk — COMMENTED** [review]
sounds good to me.

**Pull request merged**